react/backstop/data/engine_scripts/puppet/onReady.js

```diff
@@ -0,0 +1,6 @@
+module.exports = async (page, scenario, vp) => {
```

Comment:
Totally up to you, but I find having scripts in there for puppeteer, Chromy, Phantom, and Slimer is overwhelming and confusing.  We're consolidating just to a single `ready.js` and a single `before.js` in `patternlab/backstop/scripts`.